src/ITiledMap.ts

```diff
@@ -7,7 +7,7 @@
 export interface ITiledMap {
    width: number;
    height: number;
-   layers: ITiledMapLayer[];
+   layers: { [id: string] : ITiledMapLayer };
```

Comment:
I think we could support the previous method of accessing layers and the new way. See suggestions.

```suggestion
   layers: { [id: string | number] : ITiledMapLayer };
```